SU2_CFD/src/numerics/flow/convection/centered.cpp

```diff
@@ -130,6 +141,12 @@ CNumerics::ResidualType<> CCentLaxInc_Flow::ComputeResidual(const CConfig* confi
   if (variable_density) {
     MeandRhodT = -MeanDensity/MeanTemperature;
   }
+  if (energy_multicomponent) {
+    MeandRhodT /= MeanCp ;
+    MeanCp = 1.0;
+    Cp_i = 1.0;
+    Cp_j = 1.0;
+  }
```

Comment:
Documentation

Reply:
can you also add some comment above this block of code, explaining why cp=1 here?